src/host/broadcast.jl

```diff
@@ -85,7 +85,7 @@ allequal(x, y, z...) = x == y && allequal(y, z...)
 function Base.map(f, xs::AnyGPUArray...)
```

Comment:
```suggestion
function Base.map(f, x1::AnyGPUArray, xrest::AnyGPUArray...)
    xs = (x1, xrest...)
```
@maleadt on a separate note, wouldn't it be better to define the signature in this way to avoid implicitly overloading `Base.map(f)`, which is type piracy?